src/drive/trash/index.ts

```diff
@@ -44,17 +44,41 @@ export class Trash {
     return this.client.get('/storage/trash', this.headers());
   }
 
+  /**
+   * Retrieves a paginated list of trashed files or folders.
+   * @param {number} limit - The number of items to retrieve per page.
+   * @param {number} [offset=0] - The number of items to skip before beginning to return items.
+   * @param {'files' | 'folders'} type - The type of content to retrieve.
+   * @param {boolean} root - A boolean indicating whether to retrieve content from the root folder.
+   * @param {number} [folderId] - The ID of the folder to retrieve content from.
+   * @returns {Promise<FetchTrashContentResponse>} - A promise that resolves with the paginated list of trashed content.
+   */
+  public getTrashedFilesPaginated(
+    limit: number,
+    offset = 0,
+    type: 'files' | 'folders',
+    root: boolean,
```

Comment:
Add root explanation if possible. `root` param is to request to the API the root level of the trash, which could have different parent folder ids so differentiation is required.